README.md

```diff
@@ -33,7 +33,7 @@ These instructions will get you a copy of the project up and running on your loc
 
 * [Java 11](https://openjdk.java.net/projects/jdk/11/) or higher
 * [Apache Maven 3.5](https://maven.apache.org/download.cgi) or higher
-* [Lombok 1.18.16](https://projectlombok.org/download) or higher. Please check the [setup instructions](https://projectlombok.org/setup/overview) for your IDE. Note that in IntelliJ IDEA you might have to mark the `target/generated-sources/delombok` directory as a _sources_ directory and unmark/exclude `target/generated-sources/delombok`.
+* [Lombok 1.18.16](https://projectlombok.org/download) or higher. Please check the [setup instructions](https://projectlombok.org/setup/overview) for your IDE. Note that in IntelliJ IDEA you might have to mark the `src/main/lombok` directory as a _sources_ directory and _unmark_/_exclude_ `target/generated-sources/delombok`.
```

Comment:
I know that this PR is for IntelliJ IDEA but actully also on eclipse one must add a `source/main/lombok` directory as source directory. Do you want to make the small change here or should we make a new PR?

Reply:
Maybe then we could add that at another place. Maybe a separate `###` section named like IDE settings.

Reply:
> Maybe a separate ### section named something like “IDE settings”.

:+1: yeah, that would make sense.